components/collapsible-panel/README.md

```diff
@@ -0,0 +1,164 @@
+</d2l-collapsible-panel>
+
+<d2l-collapsible-panel type="subtle" panel-title="Availability Dates and Conditions">
+	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas odio ligula, aliquam efficitur sollicitudin non, dignissim quis nisl. Nullam rutrum, lectus sed finibus consectetur, dolor leo blandit lorem, vitae consectetur arcu enim ornare tortor.
+</d2l-collapsible-panel>
+
+<d2l-collapsible-panel type="inline" panel-title="Availability Dates and Conditions">
+	Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas odio ligula, aliquam efficitur sollicitudin non, dignissim quis nisl. Nullam rutrum, lectus sed finibus consectetur, dolor leo blandit lorem, vitae consectetur arcu enim ornare tortor.
+</d2l-collapsible-panel>
+```
+
+<!-- docs: start hidden content -->
+### Slots
+
+| Slot | Type | Description |
+|--|--|--|
+| `header` | optional | Supporting header content |
+| `actions` | optional | Buttons and dropdown openers to be placed in top right corner of header |
+| `summary` | optional | Summary of the expanded content. Only accepts `d2l-collapsible-panel-summary-item` |
+| `default` | required | Content that is rendered when the panel is expanded |
+
+
+### Properties
+
+| Property | Type | Description |
+|--|--|--|
+| `expanded` | Boolean | Whether or not the panel is expanded |
+| `expand-collapse-label` | String | Optional label describing the contents of the header (used by screen readers) |
+| `heading-style` | Number | The heading style to use |
+| `heading-level` | Number | Semantic heading level (h1-h4) |
+| `no-sticky` | Boolean | Disables sticky positioning for the header |
+| `padding` | String | Optionally set horizontal padding of inline panels |
+| `panel-title` | String | The title of the panel |
+| `type` | String | The type of collapsible panel |
+<!-- docs: end hidden content -->
+
+### Panel Types
+
+<!-- docs: demo live name:d2l-collapsible-panel size:large -->
+```html
+<script type="module">
+	import '@brightspace-ui/core/components/collapsible-panel/collapsible-panel.js';
+</script>
+
+<d2l-collapsible-panel panel-title="Default panel">
+	This is the default collapsible panel style
+</d2l-collapsible-panel>
+
+<d2l-collapsible-panel type="subtle" panel-title="Subtle panel">
+	This is the "subtle" collapsible panel style
+</d2l-collapsible-panel>
+
+<d2l-collapsible-panel type="inline" panel-title="Inline panel">
+	This is the "inline" collapsible panel style
+</d2l-collapsible-panel>
+```
+
+## Summary Items [d2l-collapsible-panel-summary-item]
+The summary area takes information from the expanded panel and summarizes it for the collapsed version.This can help the user understand what information is inside the panel without having to click on it.
```

Comment:
```suggestion
The summary area takes information from the expanded panel and summarizes it for the collapsed version. This can help the user understand what information is inside the panel without having to click on it.
```